src/state.rs

```diff
@@ -1,80 +1,58 @@
 use anyhow::anyhow;
-use cid::multihash::Code;
 use cid::Cid;
-use fvm_ipld_encoding::tuple::{Deserialize_tuple, Serialize_tuple};
-use fvm_ipld_encoding::{serde_bytes, to_vec, CborStore, RawBytes, DAG_CBOR};
-use fvm_ipld_hamt::{BytesKey, Error as HamtError, Hamt};
-use fvm_sdk as sdk;
-use fvm_shared::address::{Address, SubnetID};
-use fvm_shared::bigint::bigint_ser;
-use fvm_shared::bigint::bigint_ser::BigIntDe;
+use fvm_ipld_blockstore::Blockstore;
+use fvm_ipld_encoding::Cbor;
+use fvm_ipld_hamt::BytesKey;
+use fvm_shared::address::Address;
 use fvm_shared::bigint::Zero;
 use fvm_shared::clock::ChainEpoch;
 use fvm_shared::econ::TokenAmount;
-use fvm_shared::MethodNum;
-use lazy_static::lazy_static;
-use num::rational::Ratio;
+use ipc_gateway::{Checkpoint, SubnetID, DEFAULT_CHECKPOINT_PERIOD, MIN_COLLATERAL_AMOUNT};
+use primitives::{TCid, THamt};
+use serde::{Deserialize, Serialize};
 
-use fil_actor_hierarchical_sca::{Checkpoint, DEFAULT_CHECKPOINT_PERIOD, MIN_COLLATERAL_AMOUNT};
-
-use crate::blockstore::*;
 use crate::types::*;
-use crate::utils::{deserialize, ExpectedSend};
-use crate::{abort, ext};
+use crate::ExpectedSend;
 
-lazy_static! {
-    static ref VOTING_THRESHOLD: Ratio<TokenAmount> =
-        Ratio::new(TokenAmount::from(2), TokenAmount::from(3));
-}
+// lazy_static! {
+//     static ref VOTING_THRESHOLD: Ratio<TokenAmount> =
+//         Ratio::new(TokenAmount::from_atto(2), TokenAmount::from_atto(3));
```

Comment:
The goal here is to compute the `2/3` of voting power, we can probably find a more efficient way of computing this (for the next PR)